cmd/operator/generate.sh

```diff
@@ -30,9 +30,9 @@ webhook:
 EOF
 CRD="../../charts/kubearchive/crds/kubearchive.kubearchive.org_kubearchiveconfigs.yaml"
 yq -i '.metadata.annotations."cert-manager.io/inject-ca-from"="kubearchive/kubearchive-operator-certificate"' ${CRD}
-yq -i ".spec.conversion = load(\"$patch\")" ${CRD}
+yq -i ".spec.conversion = load(\"$PATCH\")" ${CRD}
 
-rm -f $patch
+rm -f $PATCH
```

Comment:
Following my previous suggestion:
```suggestion
rm -f ${PATCH}
```